config/production.yml

```diff
@@ -0,0 +1,59 @@
+custom:
+  customDomain:
+    enabled: false
+
+  resourceNames:
+    hitrate: ${env:HITRATE_TABLE}
+    credentials: ${env:CREDENTIALS_TABLE}
+    sqs: ${env:QUEUE_NAME}
+    sqsArn: ${env.QUEUE_ARN}
```

Comment:
`${env:QUEUE_ARN}` (colon instead of period)?

Reply:
Oh, yup! Good catch!